content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -0,0 +1,207 @@
+---
+reviewers:
+- liggitt
+- mikedanese
+- munnerz
+title: Certificate Signing Requests
+content_template: templates/concept
+weight: 20
+---
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+The Certificates API enables automation of
+[x509](https://tools.ietf.org/html/rfc5280) credential provisioning by providing
```

Comment:
nit:
```suggestion
[X.509](https://tools.ietf.org/html/rfc5280) credential provisioning by providing
```

X.509 is defined at https://www.itu.int/rec/T-REC-X.509 (maybe something for the What's Next section though)